app/models/miq_action.rb

```diff
@@ -730,31 +730,32 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
-  def action_container_image_annotate_deny_execution(action, rec, inputs)
+  def action_container_image_annotate_scan_results(action, rec, inputs)
+    MiqPolicy.logger.info("MIQ(#{__method__}): Now executing  [#{action.description}]")
     error_prefix = "MIQ(#{__method__}): Unable to perform action [#{action.description}], "
     unless rec.kind_of?(ContainerImage)
       MiqPolicy.logger.error("#{error_prefix} object [#{rec.inspect}] is not a Container Image")
       return
     end
 
-    unless rec.respond_to?(:annotate_deny_execution)
+    unless rec.respond_to?(:annotate_scan_policy_results)
       MiqPolicy.logger.error("#{error_prefix} ContainerImage is not linked with an OpenShift image")
       return
     end
 
     if inputs[:synchronous]
       MiqPolicy.logger.info("MIQ(#{__method__}): Now executing  [#{action.description}] for event "\
                             "[#{inputs[:event].description}]")
-      rec.annotate_deny_execution(inputs[:policy].name)
+      rec.send(:annotate_scan_policy_results, inputs[:policy].name, inputs[:result])
```

Comment:
why do we need to use `send`?

Reply:
@moolitayer  I can't remember, maybe it used to be private during some refactoring. I changed it back.